thetis/configuration.py

```diff
@@ -195,7 +195,38 @@ def info(self):
         return result + super(PairedEnum, self).info()
 
 
-class FrozenHasTraits(HasTraits):
+class OptionsBase(object):
+    """Abstract base class for all options classes"""
+
+    def name(self):
```

Comment:
I don't think that's quite right: HasTraits is not a metaclass - but it does have a custom metaclass (MetaHasTraits), so subclasses of both HasTraits and OptionsBase need a metaclass that this a subclass of both ABCMeta and MetaHasTraits. I've pushed a fix in that direction.